quimb/tensor/tensor_tebd.py

```diff
@@ -490,3 +490,107 @@ def at_times(self, ts, dt=None, tol=None, order=4, progbar=None):
                 self._set_progbar_desc(ts)
 
             yield self.pt
+
+
+def OTOC_local(psi0, H, H_back, ts, i):
+    """ The out-of-time-ordered correlator (OTOC) generating by a local
+    pauli 'z' notation acting on site 'i', note it's a function of time.
+
+    Parameters
+    ----------
+    psi0 : MatrixProductState
+        The initial state in MPS form.
+    H : NNI
+        The Hamiltonian for forward time-evolution.
+    H_back : NNI
+        The Hamiltonian for backward time-evolution, should have only
+        sign difference with 'H'.
+    ts : sequence of float
+        The time to evolve to.
+    i : int
+        The site where the local pauli 'z' notation acting on.
+    """
+
+    # obviously play with these settings
+    tebd_opts = {
+        'tol': 1e-5,
+        'split_opts': {
+            'cutoff': 1e-5,
+            'cutoff_mode': 'rel',
+        },
+    }
+
+    Z = qu.pauli('Z')
```

Comment:
Do we want to limit ourselves to 'Z' here? Might be worth having a kwarg like ``OTOC_local(..., operator='Z', ...)`` and if the argument is not a string, interpret it directly as an operator.